webapp/src/main/java/com/box/l10n/mojito/service/asset/VirtualTextUnitBatchUpdaterService.java

```diff
@@ -115,6 +115,8 @@ public void updateTextUnits(
 
     performLeveraging(savedTextUnits, nameToUsedtextUnitDTOs, contentToTextUnitDTOs);
 
+    // TODO (mallen): Schedule AI translate job asynchrously
```

Comment:
Just double checking if you intend to have this done in this PR or a subsequent one